cmd/server/assets/admin/mobileapps/show.html

```diff
@@ -0,0 +1,82 @@
+{{define "admin/mobileapps/show"}}
+
+{{$apps := .apps}}
+
+<!doctype html>
+<html lang="en">
+<head>
+  {{template "head" .}}
+</head>
+
+<body class="tab-content">
+  {{template "admin/navbar" .}}
+
+  <main role="main" class="container">
+    {{template "flash" .}}
+
+    <div class="card mb-3 shadow-sm">
+      <div class="card-header">Mobile apps</div>
+      <div class="card-body">
+
+        {{if .apps}}
+        <div class="table-responsive">
+          <table class="table table-bordered table-striped bg-white">
+            <thead>
+              <tr>
+                <th scope="col">App</th>
+                <th scope="col" width="95">Enabled</th>
+                <th scope="col" width="40"></th>
```

Comment:
I don't think we need enable/disable here 

the goal is to be able to see which mobile apps are configured for the redirect service

Reply:
sure,  removed that column